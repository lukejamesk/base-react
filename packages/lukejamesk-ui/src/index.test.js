import { testPackageExports } from '@lukejamesk/test-utils'
import * as lukejameskCore from '.'

testPackageExports('@lukejamesk/core', lukejameskCore)
