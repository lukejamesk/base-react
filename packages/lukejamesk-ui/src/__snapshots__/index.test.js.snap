// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@lukejamesk/core should match snapshot 1`] = `
Array [
  "Breakpoints",
  "ButtonBase",
  "ButtonBaseInteractor",
  "Expander",
  "ExpanderBodyInteractor",
  "ExpanderContext",
  "ExpanderHeaderInteractor",
  "ExpanderInteractor",
  "HideAt",
  "Icon",
  "ShowAt",
  "Typography",
  "library",
  "useBreakpoints",
  "useWindowSize",
]
`;
